core/src/main/scala/org/apache/spark/ui/exec/ExecutorsTab.scala

```diff
@@ -92,15 +92,18 @@ class ExecutorsListener(storageStatusListener: StorageStatusListener) extends Sp
     val info = taskEnd.taskInfo
     if (info != null) {
       val eid = info.executorId
-      executorToTasksActive(eid) = executorToTasksActive.getOrElse(eid, 1) - 1
-      executorToDuration(eid) = executorToDuration.getOrElse(eid, 0L) + info.duration
       taskEnd.reason match {
+        case Resubmitted =>
+          return
```

Comment:
I think so, because these ```Resubmitted SparkListenerTaskEnd``` tasks have post a ```Success SparkListenerTaskEnd``` before, all the metrics have been updated. If update here, the metrics will calculate twice.